azure-ts-msi-keyvault-rbac/index.ts

```diff
@@ -0,0 +1,170 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+import { execSync} from "child_process";
+
+// Create a resource group
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
+    location: "West US",
+});
+
+// Create a storage account for Blobs
+const storageAccount = new azure.storage.Account("storage", {
+    resourceGroupName: resourceGroup.name,
+    accountReplicationType: "LRS",
+    accountTier: "Standard",
+});
+
+// The container to put our files into
+const storageContainer = new azure.storage.Container("files", {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+    containerAccessType: "private",
+});
+
+// Azure SQL Server that we want to access from the application
+const sqlServer = new azure.sql.SqlServer("sqlserver", {
+    resourceGroupName: resourceGroup.name,
+
+    administratorLogin: "manualadmin",
+    administratorLoginPassword: "WeWontUseTh1sPwd",
```

Comment:
Just to encourage best practices - should we construct this as a random password using `random.RandomString`?

Reply:
Totally, great hint!